tf-modules/cumulus/ecs_cluster_instance_autoscaling_cf_template.yml.tmpl

```diff
@@ -4,6 +4,7 @@ Resources:
     Properties:
       SecurityGroups:
         - ${security_group_id}
+        - ${es_security_group_id}
```

Comment:
When passing in a security group ID, users will need to be aware that this will apply to all EC2 instances in their cluster, and will be [most permissive](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html#security-group-rules):

>If there is more than one rule for a specific port, we apply the most permissive rule. For example, if you have a rule that allows access to TCP port 22 (SSH) from IP address 203.0.113.1 and another rule that allows access to TCP port 22 from everyone, everyone has access to TCP port 22.

Perhaps the changelog note will suffice for this purpose.
We may want to consider it naming it something more descriptive, since it does not apply just to ElasticSearch now.